prisma/schema.prisma

```diff
@@ -127,6 +127,34 @@ model Claim {
   lastUpdatedAt DateTime @default(now())
 }
 
+// this is for validator service
+model CandidUserInfo {
+  id         Int    @id @default(autoincrement())
+  claimId    Int?   @unique
+  firstName  String
+  lastName   String
+  email      String
+  profileURL String
+}
+
+model Validation {
+  id        String   @id @default(uuid())
+  createdAt DateTime @default(now())
+
+  contex          String
+  validatorEmail   String
+  claimId          Int
+  validationStatus ValidationStatus @default(PENDING)
+  response         Boolean?         @default(false)
+  validationDate   DateTime?
```

Comment:
what about array of 'sentDate' might be sent more than once?  or we figure it was sent only once on the createdAt date? 

when they respond, it should not be boolean actually

the validator should create a claim using hte interface

could we have the address of the claim they created?

claimAddress

Reply:
does 'response' mean "yes they responded" or does it mean "yes they validated it positively" ?  maybe be more semantic

Reply:
since it defaults to false it must mean, they didn't respond.  i think also if there is no validationClaimId then they also didn't respond

and validationStatus also tells this

So maybe you don't need 'response' columns

Reply:
really good points, thank you

you are right we may send it more than once and we can find out if they responded or not from validationStatus we can remove that column for sure